CRM/Core/BAO/FinancialTrxn.php

```diff
@@ -484,23 +484,26 @@ public static function getPartialPaymentWithType($entityId, $entityName = 'parti
   }
 
   /**
-   * @param int $contributionId
+   * @param int $contributionID
+   * @param bool $includeRefund
    *
    * @return string
    */
-  public static function getTotalPayments($contributionId) {
-    $statusId = CRM_Core_PseudoConstant::getKey('CRM_Contribute_BAO_Contribution', 'contribution_status_id', 'Completed');
+  public static function getTotalPayments($contributionID, $includeRefund = FALSE) {
+    $statusIDs = [CRM_Core_PseudoConstant::getKey('CRM_Contribute_BAO_Contribution', 'contribution_status_id', 'Completed')];
+
+    if ($includeRefund) {
```

Comment:
Why have we split out this parameter? There are no places calling it as yet

Reply:
@eileenmcnaughton this is old followup PR of https://github.com/civicrm/civicrm-core/pull/13151 where we agreed  [here](https://github.com/civicrm/civicrm-core/pull/13151#discussion_r237800312)) to have this function modified to include refunded payments rather than only Completed. The need is to get correct contribution balance that earlier consider only Completed payments and is used in the patch at L4178 at Contribution BAO.